### PR TITLE
l10n: Changing the word "These" to "The following"

### PR DESCRIPTION
**Pull request by Valdnet**

### CODE REVIEW

File: core/templates/update.admin.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
 		<?php } ?>
 		<?php if (!empty($_['appsToUpgrade'])) { ?>
 		<div class="infogroup">
-			<span><?php p($l->t('These apps will be updated:')); ?></span>
+			<span><?php p($l->t('Following apps will be updated:')); ?></span>
```
Comment (rullzer) [inline core/templates/update.admin.php:11]: Shouldn't it be `The following` then?

Comment (Valdnet) [reply core/templates/update.admin.php:11]: @rullzer I adapted to this description:
https://github.com/nextcloud/server/blob/c6da7ec24cc913af152b4df35a52888f92a4635a/core/ajax/update.php#L231

In my opinion, "The" must be used, but it should also be changed in the latter description.
What decision?

**Review by rakekniven — CHANGES_REQUESTED** [review]
Please the "The" as start of the sentence.

**Comment by Valdnet** [comment]
@rullzer @rakekniven  OK. Changed.

**Review by rakekniven — APPROVED** [review]

**Review by jancborchardt — APPROVED** [review]

**Pull request merged**